### DISCUSSION TITLE
problem with self-service-password 1.5.3 & php-guzzlehttp-guzzle6 6.5.8 w/ php 8.2

### DISCUSSION THREAD

**Discussion by bchill** [ltb-project/self-service-password#791] [General]
Hello,

I've run into a problem with problem with self-service-password 1.5.3 & php-guzzlehttp-guzzle6 6.5.8 w/ php 8.2 that was triggered by the SMS reset function.

I had to comment out every function in...

/usr/share/self-service-password/lib/vendor/guzzlehttp/guzzle/src/functions.php

... to remove this type of error for each one:

PHP Fatal error:  Cannot redeclare GuzzleHttp\is_host_in_noproxy() (previously declared in /usr/share/self-service-password/lib/vendor/guzzlehttp/guzzle/src/functions.php:140) in /usr/share/php/GuzzleHttp6/functions.php on line 254

After commenting out those (stub?) functions, the SMS reset function worked as expected.

Am I missing something?

Brian

**Comment by coudot** [comment]
You may have GuzzleHttp already installed on your system (in /usr/share/php/GuzzleHttp6).

Is it a dependency of another PHP application?
